go/vt/vttablet/tabletmanager/rpc_vreplication_test.go

```diff
@@ -302,6 +302,7 @@ func TestCreateVReplicationWorkflow(t *testing.T) {
 // results returned. Followed by ensuring that SwitchTraffic
 // and ReverseTraffic also work as expected.
 func TestMoveTablesUnsharded(t *testing.T) {
+	t.Skip("Skipping test temporarily as it is flaky on CI, pending investigation")
```

Comment:
don't we just want to make the workflow not required?

Reply:
I just took the commit from rohit's PR. 
I do not need this change anymore as it is already in `main` now.

Making CI not required make other tests also being ignored.